antora-ui-camel/src/css/doc.css

```diff
@@ -116,7 +116,7 @@
 }
 
 .doc a {
-  color: var(--link-font-color);
+  color: #303284;
```

Comment:
We use CSS variables, it's very difficult to maintain individual values scattered across CSS files.

Reply:
oh okay, there is an option under each heading link of "continue reading " ,that link and heading link redirects us to the same page, so would it be okay to remove the link from the heading ? 
Or as in the home page the link "What's New?" has been assigned color of a regular heading 
![image](https://user-images.githubusercontent.com/44733143/77436049-d32ba980-6e04-11ea-8066-3cf45ef7e165.png)

Reply:
Oh , I'll consider this before making any further changes